<ui version="4.0" >
 <class>ParametersForm</class>
 <widget class="QWidget" name="ParametersForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>348</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>250</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Parameters</string>
  </property>
  <widget class="QGroupBox" name="leftImageBox" >
   <property name="geometry" >
    <rect>
     <x>12</x>
     <y>4</y>
     <width>139</width>
     <height>133</height>
    </rect>
   </property>
   <property name="title" >
    <string>Left Image</string>
   </property>
   <property name="flat" >
    <bool>true</bool>
   </property>
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2" >
    <item>
     <layout class="QFormLayout" name="formLayout_2" >
      <property name="fieldGrowthPolicy" >
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Offset</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSlider" name="leftOffsetSlider" >
        <property name="minimum" >
         <number>0</number>
        </property>
        <property name="maximum" >
         <number>300</number>
        </property>
        <property name="singleStep" >
         <number>1</number>
        </property>
        <property name="pageStep" >
         <number>10</number>
        </property>
        <property name="value" >
         <number>275</number>
        </property>
        <property name="sliderPosition" >
         <number>275</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance" >
         <bool>false</bool>
        </property>
        <property name="invertedControls" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Distance</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QSlider" name="leftDistanceSlider" >
        <property name="value" >
         <number>35</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Angle</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QSlider" name="leftAngleSlider" >
        <property name="minimum" >
         <number>-180</number>
        </property>
        <property name="maximum" >
         <number>180</number>
        </property>
        <property name="value" >
         <number>-10</number>
        </property>
        <property name="sliderPosition" >
         <number>-10</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="centerImageBox" >
   <property name="geometry" >
    <rect>
     <x>167</x>
     <y>4</y>
     <width>164</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title" >
    <string>Center Image</string>
   </property>
   <property name="flat" >
    <bool>true</bool>
   </property>
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <layout class="QFormLayout" name="formLayout" >
    <item row="0" column="0" >
     <widget class="QLabel" name="label_14" >
      <property name="text" >
       <string>Offset</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" >
     <widget class="QSlider" name="centerOffsetSlider" >
      <property name="minimum" >
       <number>-150</number>
      </property>
      <property name="maximum" >
       <number>150</number>
      </property>
      <property name="value" >
       <number>0</number>
      </property>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QLabel" name="label_13" >
      <property name="text" >
       <string>Distance</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QSlider" name="centerDistanceSlider" >
      <property name="minimum" >
       <number>0</number>
      </property>
      <property name="maximum" >
       <number>99</number>
      </property>
      <property name="value" >
       <number>0</number>
      </property>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0" >
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>Angle</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" >
     <widget class="QSlider" name="centerAngleSlider" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimum" >
       <number>-180</number>
      </property>
      <property name="maximum" >
       <number>180</number>
      </property>
      <property name="value" >
       <number>15</number>
      </property>
      <property name="sliderPosition" >
       <number>15</number>
      </property>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="rightImageBox" >
   <property name="geometry" >
    <rect>
     <x>348</x>
     <y>4</y>
     <width>139</width>
     <height>133</height>
    </rect>
   </property>
   <property name="title" >
    <string>Right Image</string>
   </property>
   <property name="flat" >
    <bool>true</bool>
   </property>
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3" >
    <item>
     <layout class="QFormLayout" name="formLayout_3" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Offset</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSlider" name="rightOffsetSlider" >
        <property name="maximum" >
         <number>300</number>
        </property>
        <property name="value" >
         <number>275</number>
        </property>
        <property name="sliderPosition" >
         <number>275</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Distance</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QSlider" name="rightDistanceSlider" >
        <property name="value" >
         <number>35</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>Angle</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QSlider" name="rightAngleSlider" >
        <property name="minimum" >
         <number>-180</number>
        </property>
        <property name="maximum" >
         <number>180</number>
        </property>
        <property name="value" >
         <number>10</number>
        </property>
        <property name="sliderPosition" >
         <number>10</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" native="1" name="widget" >
   <property name="geometry" >
    <rect>
     <x>12</x>
     <y>152</y>
     <width>25</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Minimum" hsizetype="Fixed" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize" >
    <size>
     <width>25</width>
     <height>15</height>
    </size>
   </property>
  </widget>
  <widget class="QGroupBox" name="reflectionBox" >
   <property name="geometry" >
    <rect>
     <x>12</x>
     <y>175</y>
     <width>139</width>
     <height>146</height>
    </rect>
   </property>
   <property name="title" >
    <string>Reflection</string>
   </property>
   <property name="flat" >
    <bool>true</bool>
   </property>
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4" >
    <item>
     <layout class="QFormLayout" name="formLayout_4" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_8" >
        <property name="text" >
         <string>Opacity</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSlider" name="reflectionOpacitySlider" >
        <property name="value" >
         <number>35</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_9" >
        <property name="text" >
         <string>Offset</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QSlider" name="reflectionOffsetSlider" >
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="value" >
         <number>35</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="backgroundBox" >
   <property name="geometry" >
    <rect>
     <x>167</x>
     <y>175</y>
     <width>165</width>
     <height>146</height>
    </rect>
   </property>
   <property name="title" >
    <string>Background</string>
   </property>
   <property name="flat" >
    <bool>true</bool>
   </property>
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_2" >
    <item row="0" column="0" >
     <widget class="QLabel" name="label_10" >
      <property name="text" >
       <string>R</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" >
     <widget class="QSlider" name="backgroundRedSlider" >
      <property name="maximum" >
       <number>255</number>
      </property>
      <property name="value" >
       <number>255</number>
      </property>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2" >
     <widget class="QSpinBox" name="redSpinBox" >
      <property name="maximum" >
       <number>255</number>
      </property>
      <property name="value" >
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QLabel" name="label_11" >
      <property name="text" >
       <string>G</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QSlider" name="backgroundGreenSlider" >
      <property name="maximum" >
       <number>255</number>
      </property>
      <property name="value" >
       <number>255</number>
      </property>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="2" >
     <widget class="QSpinBox" name="greenSpinBox" >
      <property name="maximum" >
       <number>255</number>
      </property>
      <property name="value" >
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0" >
     <widget class="QLabel" name="label_12" >
      <property name="text" >
       <string>B</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" >
     <widget class="QSlider" name="backgroundBlueSlider" >
      <property name="maximum" >
       <number>255</number>
      </property>
      <property name="value" >
       <number>255</number>
      </property>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="2" >
     <widget class="QSpinBox" name="blueSpinBox" >
      <property name="maximum" >
       <number>255</number>
      </property>
      <property name="value" >
       <number>255</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="screenie.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>backgroundRedSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>redSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>471</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel" >
     <x>531</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>backgroundRedSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>541</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel" >
     <x>409</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backgroundGreenSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>greenSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>466</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel" >
     <x>549</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>backgroundGreenSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>535</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel" >
     <x>460</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backgroundBlueSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>blueSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>467</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel" >
     <x>536</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>backgroundBlueSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>543</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel" >
     <x>474</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
